pkg/process/checks/net.go

```diff
@@ -474,3 +565,155 @@ func convertAndEnrichWithServiceCtx(tags []string, tagOffsets []uint32, serviceC
 
 	return tagsStr
 }
+
+// Manage connections that have been split into batches from the system-probe
+func processConnectionsBatch(
```

Comment:
Is this a copy of `batchConnections`? Can we look at reusing code from there?

Reply:
I have now change it to be the following flow:
`runHTTP` -> `batchConnections` -> `handleConnections` (shared) + updateConnectionsTelemetry (shared) 
`runGRPC` ->  `processConnectionsBatch` ->  `handleConnections` (shared) + `updateConnectionsTelemetry` (shared) 

My goal was not to change anything in the current HTTP flow and to only adding an optional support for the gRPC